render/render.go

```diff
@@ -240,18 +243,137 @@ func (p TextRenderer) Render(w io.Writer, x ast.Expr) (mErr error) {
 		_, mErr = w.Write([]byte("-"))
 	case *ast.TextMath:
 		if _, mErr = w.Write([]byte("$")); mErr != nil {
-			return mErr
+			return
 		}
 		if _, mErr = w.Write([]byte(t.Value)); mErr != nil {
-			return mErr
-		}
-		if _, mErr = w.Write([]byte("$")); mErr != nil {
-			return mErr
+			return
 		}
+		_, mErr = w.Write([]byte("$"))
 	case *ast.TextNBSP, *ast.TextSpace:
 		_, mErr = w.Write([]byte(" "))
+	case *ast.TextAccent:
+		var (
+			accent       accentType
+			accentedRune rune
+		)
+		if accent, mErr = newAccentType(t.Accent); mErr != nil {
+			return
+		}
+		text := t.Value.(*ast.Text).Value
+		if utf8.RuneCountInString(text) != 1 {
+			return fmt.Errorf("renderer - accent text must be a single character")
+		}
+		r, _ := utf8.DecodeRuneInString(text)
```

Comment:
Is the value always a rune? If so, let's change the type to rune instead of ast.Text.

Reply:
I've looked for a definitive answer, and it does seem that it [always a single rune](https://en.wikibooks.org/wiki/LaTeX/Special_Characters#Escaped_codes).

However, checking LaTeX's behaviour: the string `\'{ee}` is valid and parsed as `\'ee`. I've thus left the accented text to be a string and modified the rendering logic to leave other characters unchanged.